packages/wrangler/src/deployment-bundle/bindings.ts

```diff
@@ -486,7 +553,7 @@ export async function provisionBindings(
 	}
 }
 
-function getSettings(
+export function getSettings(
```

Comment:
Should this function be moved to a shared location (e.g. a file in the `utils` dir)? 🤔  (possibly as a followup)